__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+  test('GET:200 and sends array of all topics with properties of slug and description', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach((topic) => {
+          expect(typeof topic.description).toBe('string');
+          expect(typeof topic.slug).toBe('string');
+        });
+      });
+  });
+  test('GET:404 responds with a 404 status when given an invalid api route path', () => {
+    return request(app).get('/api/topiks').expect(404);
+  });
```

Comment:
Really nice test but we should also be testing the response we get back from this and this should not be inside of /api/topics and this is just a generic invalid endpoint and not related to /api/topics at all.